### PR TITLE
migrate to 16

### LINKED ISSUE #99: Upgrade to nx 16
Hey, will you please upgrade to nx16?

Let me know if you don't have time and I'm willing to do it.

### PR DESCRIPTION
**Pull request by ALMaclaine**
## Problem

not on 16

## Solution

migrate to 16

### Useful documentation
fixes #99

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,28 +15,19 @@
   "engines": {
     "npm": "please-use-pnpm",
     "yarn": "please-use-pnpm",
-    "node": "16",
+    "node": "18",
```
Comment (andreacaldera) [inline package.json:18]: Is this necessary for the migration? Some of our internal projects might still be on node 16 and I'm not sure how easily they can migrate to node 18.

**Review by andreacaldera — COMMENTED** [review]
Thanks a lot @ALMaclaine! I appreciate the effort! Just a small question about node version mainly related to internal projects

**Comment by ALMaclaine** [comment]
@andreacaldera Of course, no problem. No, it isn't necessary. Changed to '16>'.

**Review by andreacaldera — APPROVED** [review]
Thanks @ALMaclaine

**Pull request merged**